packages/serverless/src/logger.js

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
Putting it here. Same concept as our collector logger. Just with a little bit less logic.

Reply:
We can optimize the two logger.js later and create a utility for common functions. Out of this scope.